impl/ecl/src/main/java/org/wildfly/extras/transformer/eclipse/ArchiveTransformerImpl.java

```diff
@@ -124,4 +124,9 @@ private boolean transform(String[] args, boolean silent) throws IOException {
         return false;
     }
 
+    @Override
+    public boolean canTransformIndividualClassFile() {
```

Comment:
I thought this is what is meant but wasn't 100% sure before.  Thanks for changing the method name to be more clear.  I believe that the no-dependency + ASM transformers could also handle this as well but we haven't been actively working on those transformers (we could in the future).